packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -2400,6 +2400,25 @@ describe('Table', () => {
                 expect(tokenValue).toBe(expectedHeight);
             });
 
+            it('adjusts height when horizontal scrollbar is shown', async () => {
+                await element.setData(simpleTableData);
+                await waitForUpdatesAsync();
+
+                await pageObject.sizeTableToGivenRowWidth(100, element);
+                await waitForUpdatesAsync();
+
+                const tokenValue = getTableHeight();
+                const heightWithoutScrollbar = getExpectedHeight(
+                    simpleTableData.length
+                );
+                expect(parseFloat(tokenValue)).toBeGreaterThanOrEqual(
```

Comment:
nit / optional: Maybe add a comment here discussing how this case handles both overlay scrollbars and sized scrollbars for future readers.